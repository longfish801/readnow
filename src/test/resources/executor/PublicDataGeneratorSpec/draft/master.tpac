#! master
#>

#> pubyears
2022
#>

#> authors
#>

#>> author:方丈貴恵
#-hiraName ★
#>

#>> author:山沢晴雄
#-hiraName ★
#>

#>> author:戸田和光
#-hiraName ★
#>

#>> author:本格ミステリ作家クラブ
#-hiraName ★
#>

#> tags
#>

#>> tag:特殊設定ミステリ
#>

#>> tag:〈竜泉家の一族〉シリーズ
#>

#>> tag:アリバイ崩し
#>

#>> tag:アンソロジー
#>

#>> tag:私的ベスト2022
#>

#> categories
#>

#>> category:趣向／サブジャンル
特殊設定ミステリ
アリバイ崩し
#>

#>> category:形態
アンソロジー
#>

#>> category:シリーズ
〈竜泉家の一族〉シリーズ
#>

#>> category:私的ベスト
私的ベスト2022
#>

#>

#!

