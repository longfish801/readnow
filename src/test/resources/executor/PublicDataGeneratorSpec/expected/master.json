{
    "pubyears": [
        "2022"
    ],
    "authors": {
        "0001": {
            "id": "0001",
            "name": "方丈貴恵",
            "hiraName": "★",
            "reviews": [
                "20220101"
            ]
        },
        "0002": {
            "id": "0002",
            "name": "山沢晴雄",
            "hiraName": "★",
            "reviews": [
                "20220201"
            ]
        },
        "0003": {
            "id": "0003",
            "name": "戸田和光",
            "hiraName": "★",
            "reviews": [
                "20220201"
            ]
        },
        "0004": {
            "id": "0004",
            "name": "本格ミステリ作家クラブ",
            "hiraName": "★",
            "reviews": [
                "20220601"
            ]
        }
    },
    "tags": {
        "0001": {
            "id": "0001",
            "name": "特殊設定ミステリ",
            "reviews": [
                "20220101"
            ]
        },
        "0002": {
            "id": "0002",
            "name": "〈竜泉家の一族〉シリーズ",
            "reviews": [
                "20220101"
            ]
        },
        "0003": {
            "id": "0003",
            "name": "アリバイ崩し",
            "reviews": [
                "20220201"
            ]
        },
        "0004": {
            "id": "0004",
            "name": "アンソロジー",
            "reviews": [
                "20220601"
            ]
        },
        "0005": {
            "id": "0005",
            "name": "私的ベスト2022",
            "reviews": [
                "20220101"
            ]
        }
    },
    "categories": [
        {
            "id": "0001",
            "name": "趣向／サブジャンル",
            "tags": [
                "0001",
                "0003"
            ]
        },
        {
            "id": "0002",
            "name": "形態",
            "tags": [
                "0004"
            ]
        },
        {
            "id": "0003",
            "name": "シリーズ",
            "tags": [
                "0002"
            ]
        },
        {
            "id": "0004",
            "name": "私的ベスト",
            "tags": [
                "0005"
            ]
        }
    ],
    "pubdates": {
        "2022": {
            "202201": [
                "20220101"
            ],
            "202202": [
                "20220201"
            ],
            "202206": [
                "20220601"
            ]
        }
    }
}